client/src/main/com/sinch/sdk/domains/verification/adapters/converters/VerificationsDtoConverter.java

```diff
@@ -69,207 +88,401 @@ public static InitiateVerificationResourceDto convert(StartVerificationRequestPa
+      }
+      if (flashCallDto.getDenyCallAfterDefined()) {
+        builder.setDenyCallAfter(flashCallDto.getDenyCallAfter());
+      }
+    }
+    return builder;
+  }
+
+  static StartVerificationResponseCallout.Builder convert(
+      CalloutInitiateVerificationResponseDto dto) {
+
+    StartVerificationResponseCallout.Builder builder = StartVerificationResponseCallout.builder();
+
+    if (dto.getIdDefined()) {
+      builder.setId(VerificationId.valueOf(dto.getId()));
+    }
+    if (dto.getLinksDefined()) {
+      builder.setLinks(LinkDtoConverter.convert((dto.getLinks())));
+    }
+    return builder;
+  }
+
+  static StartVerificationResponseSeamless.Builder convert(
+      DataInitiateVerificationResponseDto dto) {
+
+    StartVerificationResponseSeamless.Builder builder = StartVerificationResponseSeamless.builder();
+
+    if (dto.getIdDefined()) {
+      builder.setId(VerificationId.valueOf(dto.getId()));
+    }
+    if (dto.getLinksDefined()) {
+      builder.setLinks(LinkDtoConverter.convert((dto.getLinks())));
+    }
+
+    if (dto.getSeamlessDefined()) {
+      DataInitiateVerificationResponseSeamlessDto seamlessDto = dto.getSeamless();
+      if (seamlessDto.getTargetUriDefined()) {
+        builder.setTargetUri(seamlessDto.getTargetUri());
+      }
+    }
+    return builder;
   }
 
   public static VerificationReportRequestResourceDto convert(
       VerificationReportRequestParameters client) {
 
     VerificationReportRequestResourceDto dto = new VerificationReportRequestResourceDto();
 
-    client
-        .getMethod()
-        .ifPresent(
-            f -> dto.method(VerificationMethodDto.fromValue(EnumDynamicConverter.convert(f))));
-
     if (client instanceof VerificationReportFlashCallRequestParameters) {
       VerificationReportFlashCallRequestParameters typedClient =
           (VerificationReportFlashCallRequestParameters) client;
-      dto.flashcall(convert(typedClient));
+      dto.setActualInstance(convert(typedClient));
     } else if (client instanceof VerificationReportSMSRequestParameters) {
       VerificationReportSMSRequestParameters typedClient =
           (VerificationReportSMSRequestParameters) client;
-      dto.sms(convert(typedClient));
+      dto.setActualInstance(convert(typedClient));
     } else if (client instanceof VerificationReportCalloutRequestParameters) {
       VerificationReportCalloutRequestParameters typedClient =
           (VerificationReportCalloutRequestParameters) client;
-      dto.callout(convert(typedClient));
+      dto.setActualInstance(convert(typedClient));
     }
     return dto;
   }
 
-  private static VerificationReportRequestResourceFlashcallDto convert(
+  private static FlashcallVerificationReportRequestDto convert(
       VerificationReportFlashCallRequestParameters client) {
 
-    FlashcallVerificationReportRequestDto dto = new FlashcallVerificationReportRequestDto();
+    FlashcallVerificationReportRequestFlashCallDto subfield =
```

Comment:
Exact (and uncle-B will agree with this)
Because of I'm working onto migration to new generator tool usage there will be a lot of conflict in this file.
I'm doing the modification onto this incoming PR, still for 1.1. release.
So won't be part of this PR, but will be done for 1.1  
Thank you to have highlighted this.